R/aggregateSolute.R

```diff
@@ -103,17 +105,17 @@
 #' @export
 aggregateSolute <- function(
   preds, metadata, format=c("conc", "flux rate", "flux total"), 
-  agg.by=c("unit", "day", "month", "water year", "calendar year", "total", "[custom]"),
+  agg.by=c("unit", "day", "month", "water year", "calendar year", "total", "all years","[custom]"),
   se.preds, dates, custom=NA, 
   cormat.function=cormat1DayBand,
   ci.agg=TRUE, level=0.95, deg.free=NA, ci.distrib=c("lognormal","normal"), se.agg=TRUE,
-  na.rm=FALSE, attach.units=FALSE) {
+  na.rm=FALSE, attach.units=FALSE, complete.threshold = 0) {
   
   # Validate arguments
   format <- match.arg.loadflex(format, c("conc", "flux rate", "flux total"))
   attach.units <- match.arg.loadflex(attach.units)
   for(abi in 1:length(agg.by)) {
-    agg.by[abi] <- match.arg.loadflex(agg.by[abi], c("unit", "day", "month", "water year", "calendar year", "total", colnames(custom)))
+    agg.by[abi] <- match.arg.loadflex(agg.by[abi], c("unit", "day", "month", "water year", "calendar year", "total", "all years", colnames(custom)))
```

Comment:
I'm working on `all_years`, but if we want to do it differently/let the user aggregate thats fine